lib/sacastats_web/templates/character/weapons.html.heex

```diff
@@ -74,7 +74,7 @@
                         <img src={"https://census.daybreakgames.com/files/ps2/images/static/" <> (SacaStatsWeb.CharacterController.get_medal_code(SacaStatsWeb.CharacterController.get_total_values(Map.get(weapon, "weapon_kills"), @character["response"]["faction_id"]), 
                             SacaStatsWeb.CharacterController.get_total_values(Map.get(weapon, "weapon_vehicle_kills"), @character["response"]["faction_id"]))|> Integer.to_string()) <>".png"} alt="medal" class="medal float-left" width="30px"/>
                     <% end %>
-                    <img src="https://census.daybreakgames.com/files/ps2/images/static/2021.png" alt={Map.get(weapon, "name", "N/A")} class="weapon mx-auto d-block" width="150px"/>
+                    <img src={"https://census.daybreakgames.com" <> Map.get(weapon, "image_path", "/files/ps2/images/static/3.png")} alt={Map.get(weapon, "name", "N/A")} class="weapon mx-auto d-block" width="150px"/>
                     <h5 class="weaponName align-text-bottom text-center mb-0"><%=Map.get(weapon, "name", "N/A")%></h5>
                     <div id={"weapon" <> (weapon_id |> Integer.to_string()) <> "ProgressBar"} class="progress-bar progress-bar-striped progress-bar-animated aurax-progress" role="progressbar" 
```

Comment:
Is this concatenation being removed?

Reply:
It's being added I think. Git did this diff a bit weirdly

Reply:
just woke up and I now actually see what you're talking about. It looks like GitHub thinks `<>` is an invalid HTML tag instead of Elixir, since there's no diff indicator on the left